uk.co.compucorp.civicrm.hrleaveandabsences/views/shared/components/leave-request-popup/details-tab/by-unit-and-days-mode/partials/hours-unit-from-deduction.html

```diff
@@ -0,0 +1,19 @@
+<div class="row" ng-if="!detailsTab.isLeaveType('toil')">
```

Comment:
Future wishlist: create a decorator component that accepts a time from OR time to and takes care of all this so it can be reusable for either case.

Reply:
Yes, this is planned.